src/client/components/Table/Table.tsx

```diff
@@ -1,9 +1,21 @@
 import { Table, Skeleton } from '@mantine/core'
 import classes from './Table.module.css'
 import cx from 'clsx'
+import z from '../../assets/zerg.svg'
+import t from '../../assets/terran.svg'
+import p from '../../assets/protoss.svg'
+import r from '../../assets/random.svg'
 import { getLeagueSrc } from '../../utils/rankingHelper'
 
 export function RankingTable({ data, loading }) {
+
```

Comment:
I think this code would be better placed on the constants folder to keep the table component small